apartmenttherapy/oxygen-lib-ui
# Shannon/accessibility improvements

#### What does this PR do & why?
- give all buttons and input elements aria labels so they can be used by accessibility devices

#### How can this be manually tested?
- go to any page locally, here are some :-D
  - http://localhost:8080/
  - http://localhost:8080/how-to-make-easy-tomato-pie-260388 (recipe and slideshow)
  - http://localhost:8080/this-is-the-best-tahini-you-can-buy-in-the-states-239941 (product component and pin it buttons)
  - http://localhost:8080/collection/bundle-samples (scroll while lighthouse is loading to get full page audit, most layouts)
- inspect element and go to audits tab
- make sure Accessibility checkbox is checked in the audits area, (even better, uncheck everything else)
- scroll down, click the radio button for 'no throttling'
- click 'run audits'
  - you must click the 'clear all' stop button at the top left and then run audit for each page, will not auto reload.
- see that each page's accessibility rating is 94

#### Any questions or concerns about this change?

_(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)_

#### Deployment Notes

_(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)_

#### Link to story card and other documentation
https://app.asana.com/0/inbox/625204677761211/850531091458563/848571872089558

#### GIF

#### Have you remembered to...?
